.gitignore

```diff
@@ -1,4 +1,3 @@
 node_modules
-dist
```

Comment:
That is bad idea :)
We are happy to review once `dist` folder is removed and entry is put back into a gitignore